src/classes/metadata/slides_notes.vala

```diff
@@ -1,11 +1,11 @@
 /**
- * Storage for the notes of a presentation
+ * Storage for the jump points
```

Comment:
Oh that is just the second commit. I forgot to change this line in my initial commit (see ba4323f). Have a look at the first commit, it contains the actual changes.